src/tests/test_common/homestore_test_common.hpp

```diff
@@ -359,6 +359,14 @@ class HSTestHelper {
             // Fake restart, device list is unchanged.
             shutdown_homestore(false);
             std::this_thread::sleep_for(std::chrono::seconds{shutdown_delay_sec});
+        } else if (!m_token.devs_.empty()) {
```

Comment:
now we have three use cases, priority-wise:
1. UT specificly passed dev_list, if not there, 
2. --device_list from input, if not there, 
3. m_token.name_ which equeals to test_binary, name. 

Can we put it as in comment at line: 178, so that it can be friendly to us after a few month when we look back and don't need to scratch our head wondering why?